src/main/java/raisetech/student/management/validation/ValidSex.java

```diff
@@ -0,0 +1,24 @@
+package raisetech.student.management.validation;
+
+import jakarta.validation.Constraint;
+import jakarta.validation.Payload;
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+import javax.management.ConstructorParameters;
+
+@Documented
+@Constraint(validatedBy = SexValidator.class)
+@Target({ElementType.PARAMETER, ElementType.FIELD})
+@Retention(RetentionPolicy.RUNTIME)
+public @interface ValidSex {
+
+  String message() default "無効な入力です。「男性」「女性」「その他」のいずれかを入力してください。";
```

Comment:
memo
ここで無名関数呼び出しで message のデフォルト設定できたらいいのにな。
```suggestion
String message() default () -> {
  return "無効な入力です。「%s」「%s」「%s」のいずれかを入力してください。".formatted(SexValidator.Sex.男性, SexValidator.Sex.女性, SexValidator.Sex.その他);
}
```
これだともし表示ラベルが変わっても影響範囲がすくない。
for 文使えばもし表示ラベルが増えても対応できる。

が、 Java の言語仕様上これはできないのでただのぼやきです。